### DISCUSSION TITLE
Forbidden access when changing monitoring-path

### DISCUSSION THREAD

**Discussion by LeonardoIPM** [javamelody/javamelody#1218] [Q&A]
I have a Spring 3 application using JavaMelody and Prometheus.

Tried to change my property javamelody.init-parameters.monitoring-path, but when I try to access the new url, I get 403 error.
If this property isn't changed, then the ip:port/actuator/monitoring works perfectly. (I tried to remove actuator and prometheus, and ip:port/monitoring works perfectly also, but when I change it to another path, then I get 403).

Is there another configuration that I need to set? I'm not using Spring Security

**Comment by evernat** [comment]
It seems that you have closed the question. So you probably found the answer.
